### PR TITLE
New extension: New commands for Stripe

### PR DESCRIPTION
**Pull request by pradeepb28**
## Description

This extension allows you to make an invoice, create a payment, add a new product, and start a new subscription.

## Screencast
CleanShot 2022-03-10 at 09 05 44@2x

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed in the `media` folder

### CODE REVIEW

**Comment by pradeepb28** [comment]
@pernielsentikaer looking for some help, I am not quite sure on the following error. I mean what is up with description property has to with a specific match pattern. Any help is appreciated :)

CleanShot 2022-03-10 at 09 21 55@2x

**Review by fe9lix — COMMENTED** [review]
Hey @pradeepb28, thanks for your contribution. I was wondering if Raycast's built-in Quicklinks feature could replace this extension? Using Quicklinks, you can create custom items in root search and they can open any URL with any application that you choose (check out the "Create Quicklink" command).

**Comment by pradeepb28** [comment]
@fe9lix Thanks, I will check it out. Meanwhile I have a follow up question, is it possible for other users to install quicklinks ?(Just like how extensions are installed)

**Comment by fe9lix** [comment]
> @fe9lix Thanks, I will check it out. Meanwhile I have a follow up question, is it possible for other users to install quicklinks ?(Just like how extensions are installed)

Nope, not yet – but sharing of Quicklinks is something that we plan to add in the future.

File: extensions/stripe/package.json [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  "name": "stripe",
+  "title": "Stripe",
+  "description": "This extension allows you to make an invoice, create a payment, add a new product, and start a new subscription.",
+  "icon": "stripe.png",
+  "author": "pradeepb28",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "create-a-new-invoice",
+      "title": "Create a new invoice",
```
Comment (fe9lix) [inline extensions/stripe/package.json:12]: All command titles should use title case ("Create a New Invoice" or just "Create New Invoice"), please also see the [store guidelines on this](https://developers.raycast.com/basics/prepare-an-extension-for-store#extensions-and-commands-naming)

**Comment by pradeepb28** [comment]
>Nope, not yet – but sharing of Quicklinks is something that we plan to add in the future.

Would it be ok if you guys allow this an extension for now and later I can move this to quicklinks?

File: extensions/stripe/package.json [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+      "subtitle": "Stripe",
+      "description": "It creates an invoice for your business on your stripe account",
+      "mode": "view"
```
Comment (fe9lix) [inline extensions/stripe/package.json:15]: You can set all command to mode "no-view" since otherwise Raycast opens a view and shows the loading indicator.

**Comment by fe9lix** [comment]
> Would it be ok if you guys allow this an extension for now and later I can move this to quicklinks? @fe9lix

In that case, would it be more suitable to rename the extension to something like "stripe-links" or are you planning on adding more features to the extension?

I've left two comments regarding the title and mode of the commands.

File: extensions/stripe/package.json (hunk 1 above)
Comment (PitNikola) [reply extensions/stripe/package.json:12]: Drop the articles, they are redundant in titles like this

**Comment by pradeepb28** [comment]
>In that case, would it be more suitable to rename the extension to something like "stripe-links" or are you planning on adding more features to the extension?
Yes, in the next iteration I will be using Stripe API to get data

**Comment by fe9lix** [comment]
@pradeepb28 If we keep it as a links-only extension, some points to be addressed are:

- Update the command titles to use the title case convention
- Set the command modes to "no-view"
- Update the extension title to something like "Stripe Quicklinks"

**Comment by pradeepb28** [comment]
@fe9lix done

**Comment by fe9lix** [comment]
@pradeepb28 As Petr mentioned above, can you please also remove the "a" in all command titles? ("Create New Invoice" etc. is enough)

**Comment by pradeepb28** [comment]
@fe9lix done, removed "a".

**Review by fe9lix — APPROVED** [review]

**Pull request merged**